Files/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -57,6 +72,12 @@ private async void OpenInNewWindow_Click(object sender, RoutedEventArgs e)
             await Interaction.OpenPathInNewWindowAsync(item.Path);
         }
 
+        private async void OpenDriveProperties_Click(object sender, RoutedEventArgs e)
+        {
+            var item = ((MenuFlyoutItem)sender).DataContext as DriveItem;
+            await new Interaction(associatedInstance).OpenPropertiesWindowAsync(item);
```

Comment:
`AppInstance.InteractionOperations.OpenPropertiesWindowAsync(item);`?

Reply:
Will do.

Reply:
Nvm, keep it as it is.

Reply:
Oh, okay. I was just about to test and commit. I'll keep it as it is.